SU-HSDO/suhumsci
# Shs 111 table view header fix

## Summary
This fixes an error where all JavaScript would break when table view headers were not present.

## Steps to Test
1. Create a flexible page with a view that has table headers. I chose 'Default Courses' as I was able to replicate the bug.
2. View the page and make sure the table headers are present, there's no console errors related to the table and JS still works (navigation dropdown for example).
3. Go into the 'Default Courses' view, remove the headers (screenshot below), click apply all and save the view. View the page you created with that view and confirm the headers don't show up, there's no console errors related to the table and JS still works (navigation dropdown for example).

## Notes
- After exploring `table-pattern.js`, I'm not sure what that file is even doing. If I remove the contents of the entire file, the page, table, and headers still work. I think it may be a useless file, but I don't want to remove it given the fragility of files like this. 

## Screenshots
![Screen Shot 2022-09-26 at 1 12 37 PM](https://user-images.githubusercontent.com/8405274/192347120-b38d44dd-6a9a-4b35-bc20-e04528cfcc29.png)

![Screen Shot 2022-09-26 at 1 52 10 PM](https://user-images.githubusercontent.com/8405274/192347136-a1e309b0-e363-40de-b1b2-20014ed5248c.png)
